demo/app.py

```diff
@@ -138,9 +115,18 @@ def gen_button_clicked():
     )
     st.divider()
 
-    system_prompt = st.text_area("Podcast generation prompt", value=PODCAST_PROMPT)
+    st.subheader("Speaker configuration")
+    for s in DEFAULT_SPEAKERS:
+        s.pop("id", None)
+    speakers = st.data_editor(DEFAULT_SPEAKERS, num_rows="dynamic")
```

Comment:
I am getting this here... 
```
/demo/app.py", line 121, in <module>
    speakers = st.data_editor(DEFAULT_SPEAKERS, num_rows="dynamic")
               ^^^^^^^^^^^^^^
AttributeError: module 'streamlit' has no attribute 'data_editor'
```

this is due to my streamlit version `1.22.0`. We should probably pin it in `pyproject.toml`. I guess 1.41.0 (latest) is the safest? @daavoo

Reply:
There is some sort of conflict with parler which was failing when pinning.

In the .github/setup.sh i worked around the issue by running `pip install --upgrade streamlit` at the very end.

I will try to review this after we put outetts as the default requirement